ghostferry_db.go

```diff
@@ -0,0 +1,44 @@
+package ghostferry
+
+import (
+	"context"
+	"database/sql"
+)
+
+type GhostferryDB struct {
```

Comment:
Is `ghostferry.GhostferryDB` a bit verbose? Maybe just DB is fine?

Reply:
Or DBWrapper to differentiate it from sql.DB. I don't have a strong preference here.